### DISCUSSION TITLE
tflint without terraform init

### DISCUSSION THREAD

**Discussion by h2oearth** [terraform-linters/tflint#2013] [Q&A]
Hi,
We are using tflint in our GitHub pipeline by running the following command: `--module --color   --recursive -f compact`. The issue is that we are not able to run `terraform init` beforehand, and if we run `tflint --init`, this command expects that we run `terraform init` first.
Is it possible to run `tflint` without running `terraform init`?

this is our `tflint.hcl`
```
plugin "aws" {
    enabled = true
    version = "0.30.0"
    source  = "github.com/terraform-linters/tflint-ruleset-aws"
}
```

**Comment by bendrucker** [comment]
https://github.com/terraform-linters/tflint/blob/master/docs/user-guide/calling-modules.md

> By default, TFLint only calls local modules whose the source is a relative path like ./*. If you want to call remote modules (registry, git, etc.), you must run terraform init (or terraform get) before invoking TFLint so that modules are loaded into the .terraform directory. After that, invoke TFLint with --call-module-type=all

https://github.com/terraform-linters/tflint/blob/23eb00864ea39143ac869d223d185b55b32b2008/cmd/option.go#L61-L64

<hr>

Remove `--module`. By default TFLint now visits local child modules. You only need to `terraform init` first with `--call-module-type all` which also considers modules from remote sources.
